frontend/components/Dashboard.tsx

```diff
@@ -7,31 +7,29 @@ import DuccStatsList, { ScoreType } from "./DuccStatsList";
 
 export default function Dashboard() {
   return (
-    <div className={"dashboard-container"}>
+    <div className={"dashboard-container"} style={{ fontSize: "1.2em" }}>
```

Comment:
Why put the font size here and not within the CSS where the dashboard-container class is defined?

Reply:
As I see it it's defined in Dashboard.scss as well, but with a different value.

Reply:
removed, not necessary as I was only testing and forget to delete